### PR TITLE
Use triple-double-quotes for docstrings

### PR DESCRIPTION
**Pull request by timhoffm**

### CODE REVIEW

File: doc/api/next_api_changes/removals.rst [hunk 1]
```diff
@@ -51,6 +51,8 @@ Classes and methods
 
 - ``sphinxext.plot_directive.plot_directive()``
   (use the class ``PlotDirective`` instead)
+- ``sphinxext.mathmpl.math_directive()``
```
Comment (QuLogic) [inline doc/api/next_api_changes/removals.rst]: Seem unrelated?

Comment (timhoffm) [reply doc/api/next_api_changes/removals.rst]: Sorry, messed up my branches. Fixed.

File: lib/matplotlib/mathtext.py [hunk 2]
```diff
@@ -101,7 +101,7 @@ def __init__(self):
         self.depth = 0
 
     def set_canvas_size(self, w, h, d):
-        'Dimension the drawing canvas'
+        """Set the dimension the of drawing canvas."""
```
Comment (anntzer) [inline lib/matplotlib/mathtext.py]: typo "of the"

File: lib/matplotlib/offsetbox.py [hunk 3]
```diff
@@ -713,25 +713,20 @@ def get_offset(self):
...
         ox, oy = self.get_offset()  # w, h, xd, yd)
 
         return mtransforms.Bbox.from_bounds(ox - xd, oy - yd, w, h)
 
     def get_extent(self, renderer):
-        """
-        Return with, height, xdescent, ydescent of box
-        """
-
+        """Return with, height, xdescent, ydescent of box."""
```
Comment (anntzer) [inline lib/matplotlib/offsetbox.py]: width

File: lib/matplotlib/pylab.py [hunk 4]
```diff
@@ -229,8 +229,8 @@
 ## We are still importing too many things from mlab; more cleanup is needed.
 
 from matplotlib.mlab import (
-    demean, detrend, detrend_linear, detrend_mean, detrend_none,
```
Comment (anntzer) [inline lib/matplotlib/pylab.py]: unrelated?
